source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACRToggleInputDataSource.mm

```diff
@@ -7,35 +7,41 @@
 
 #import <Foundation/Foundation.h>
 #import "ACRToggleInputDataSource.h"
-#import "ToggleInput.h"
 #import "ACRIBaseCardElementRenderer.h"
 #import "HostConfig.h"
+#import "ACRUILabel.h"
+#import "ACRColumnSetView.h"
 
 using namespace AdaptiveCards;
 
 @implementation ACRToggleInputDataSource
 {
-    std::shared_ptr<ToggleInput> toggleInputDataSource;
-    std::shared_ptr<HostConfig> config;
+    std::shared_ptr<ToggleInput> _toggleInputDataSource;
+    std::shared_ptr<HostConfig> _config;
+    UISwitch *_toggleSwitch;
+    NSString *_title;
 }
 
 - (instancetype)initWithInputToggle:(std::shared_ptr<ToggleInput> const&)toggleInput
       WithHostConfig:(std::shared_ptr<HostConfig> const&)hostConfig
 {
     self = [super init];
+    _title = [NSString stringWithCString:toggleInput->GetTitle().c_str()
+                                encoding:NSUTF8StringEncoding];
+    _toggleSwitch = [[UISwitch alloc] init];
```

Comment:
These assignments should go inside the `if (self)` block. If self is nil (unlikely though that may be) we don't want to segfault.